### DISCUSSION TITLE
2.8.8 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4858] [讨论]
· 新主页预设：PCL 更新日志（#4845，由 @Joker2184 制作）
· 优化许可与版权声明的排版（#4832，@Joker2184）
· 帮助库编修（Help#388~391，PR @JingHai-Lingyun、@Ad-closeNN）
· 更新 9 月赞助者列表，感谢大家的支持！
· 修复了以下 3 个与新正版登录相关的 Bug
　　需要使用密码登录时，提示登录轮询失败（#4828，@wanweishe）
　　在登录时按回车或 Esc 导致崩溃（#4857，@wuliaodexiaoluo）
　　若出现网络波动，理论上可以导致不应出现的登录失败
· 修复了以下 7 个其他的 Bug
　　取消更新 Mod 可能导致更新流程无法结束（#4310，@Louis-Harsune）
　　在特定情况下，加载器任务会错误地提前启动（#4835，@lactobionicAcid）
　　缓存文件夹路径存在中文可能导致出错或崩溃（#4849，@sadfadwe）
　　Mod 管理中，特定名字长度的 Mod 会导致极端卡顿（#4854，@Raisel0）
　　Mod 管理中，在搜索后先启用/禁用再删除 Mod 导致出错（#4855，@SweeCandy）
　　有多个相似的下载任务时，理论上可能出现异常
　　下载源返回 Too Many Request 会被错误地禁用

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.8.8'>2.8.8</a>.</em>

**Comment by Joker2184** [comment]
更新了.jpg
龙猫不睡觉吗.png

**Reply by iPodtouch6557** [reply]
戳啦，是劳模啦
